stable/postgresql/templates/statefulset.yaml

```diff
@@ -251,7 +250,4 @@ spec:
         storageClassName: "{{ .Values.persistence.storageClass }}"
       {{- end }}
       {{- end }}
-{{- else }}
```

Comment:
I don't understand why you disable the `emptyDir` when `persistence.enabled == false`

Reply:
because it was not mounted anyway ... 🤷‍♂️ .
this is the full diff between the statefulset created from chart v3.0.0 and my branch version:

```
$ diff $HOME/Downloads/tests/v3_results_master.yaml $HOME/Downloads/tests/v3_results_fix.yaml
85,86d84
<       - name: data
<         emptyDir: {}
```